website/docs/user-guide/concepts/external-services.md

```diff
@@ -0,0 +1,112 @@
+# External Services
+
+This document describes how to configure credentials for external services, like source code or artifact repositories.
+
+## Infrastructure Services
+
+External services are always accessed via _Infrastructure Services_.
+An Infrastructure Service can be created via API/UI or defined in a project's `.ort.env.yml`. It has the following properties:
+
+- An identifying `name`
+- An `url` to the external service
+- An optional `description`
+- A `username secret reference`
+- A `password secret reference`
+- Optional `credentials types`
+
+Please note, that you have to set a username secret reference, even if it is not needed for your service because you e.g. use a PAT.
```

Comment:
Nits:
* No comma after "note"
* "...for your service, for example, because you use a PAT."
* In general, use impersonal speech like "Please note that the username secret reference must always be set, even..."